index.html

```diff
@@ -42,6 +42,202 @@ <h4> I’am a Software Developer</h4>
+            <a class="see_project2" href="#">See Project</a>
+        </section>
+
+        <section class="card1">
+            <h2>Profesional Art Printing Data</h2>
+            <p class="jobs_p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard
+            </p>
+            <div class="skills1">
+                <ul>
+                    <li>html</li>
+                    <li>javascript</li>
+                    <li>ruby</li>
+                </ul>
+            </div>
+            <a class="see_project2" href="#">See Project</a>
+        </section>
+
+        <section class="card1">
+            <h2>Profesional Art Printing Data</h2>
+            <p class="jobs_p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard
+            </p>
+            <div class="skills1">
+                <ul>
+                    <li>html</li>
+                    <li>javascript</li>
+                    <li>ruby</li>
+                </ul>
+            </div>
+            <a class="see_project2" href="#">See Project</a>
+        </section>
+
+        <section class="card1">
+            <h2>Profesional Art Printing Data</h2>
+            <p class="jobs_p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard
+            </p>
+            <div class="skills1">
+                <ul>
+                    <li>html</li>
+                    <li>javascript</li>
+                    <li>ruby</li>
+                </ul>
+            </div>
+            <a class="see_project2" href="#">See Project</a>
+        </section>
+
+        <section class="card1">
+            <h2>Profesional Art Printing Data</h2>
+            <p class="jobs_p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard
+            </p>
+            <div class="skills1">
+                <ul>
+                    <li>html</li>
+                    <li>javascript</li>
+                    <li>ruby</li>
+                </ul>
+            </div>
+            <a class="see_project2" href="#">See Project</a>
+        </section>
+
+        <section class="card1">
+            <h2>Profesional Art Printing Data</h2>
+            <p class="jobs_p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard
+            </p>
+            <div class="skills1">
+                <ul>
+                    <li>html</li>
+                    <li>javascript</li>
+                    <li>ruby</li>
+                </ul>
+            </div>
+            <a class="see_project2" href="#">See Project</a>
+        </section>
```

Comment:
- [ ] Kindly use `div tag` instead of section tags for each project card, this will make your website more semantic and easily readable by browsers 👍🏿  
- [ ] Please follow the button styling for: Active, Hover and Pressed, see screenshot below ⬇️ 


![uuuuuuu](https://user-images.githubusercontent.com/69638013/159150728-bf567afc-f07c-4b05-95b9-e1403a7c6d38.png)